src/main/java/org/gridsuite/study/server/repository/ImportParametersEntity.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.study.server.repository;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Entity
+@Table(name = "importParameters")
+public class ImportParametersEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private UUID id;
+
+    @ElementCollection
+    @CollectionTable
```

Comment:
To change the table name specify a value or remove this annotation

Reply:
Done